### PR TITLE
CoC committe information

### PR DESCRIPTION
**Pull request by theRealImy**
As per the last Solid Team meeting a CoC committee was formed. Information was updated accordingly.

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Code of Conduct
 
-Personal threats or discriminatory threats towards arbitrary groups on Solid gitter, Solid forum, or the W3C Solid Community Group call and mailing list may result in a ban of the threatening individual by the Solid Manager.
+Personal threats or discriminatory threats towards arbitrary groups on Solid gitter, Solid forum, or the W3C Solid Community Group call and mailing list may result in a ban of the threatening individual by the CoC committee.
```
Comment (csarven) [inline code-of-conduct.md:3]:
```suggestion
Personal threats or discriminatory threats towards arbitrary groups on Solid gitter, Solid forum, or the W3C Solid Community Group call and mailing list may result in a ban of the threatening individual by the Code of Conduct (CoC) committee.
```

File: code-of-conduct.md [hunk 2]
```diff
@@ -27,16 +27,16 @@ Best practices for the Solid Community include:
...
 - Maintain an open mindset and encourage others to do the same. There might be things you or others haven't heard about yet, so be open to send and receive pointers to more info.
 - There are multiple ways of achieving the same goals, but when you build something new, check what already exists so we can better understand and help you with your contribution.
 
 ## Reporting a Code of Conduct Concern
 To report a code of conduct concern you may:
-- [Submit an issue](https://github.com/solid/culture/issues/new) to the solid/culture GitHub repository
-- Contact the Solid Manager via gitter
-- Email [manager@solidproject.org](mailto:manager@solidproject.org)
-- Write to the [solid/chat](https://gitter.im/solid/chat) room on gitter with your concern
+- Email/contact any members of the CoC committee: [Osmar Olivo](https://gitter.im/oolivo) oz@inrupt.com, [Virginia Balseiro](https://gitter.im/VirginiaBalseiro) virginiabalseiro@gmail.com, [Alain Bourgeois](https://gitter.im/bourgeoa) alain.bourgeois10@gmail.com. Committee members will recuse
```
Comment (csarven) [inline code-of-conduct.md:36]:
```suggestion
- Email/contact any members of the CoC committee: [Osmar Olivo](https://gitter.im/oolivo) oz@inrupt.com, [Virginia Balseiro](https://gitter.im/VirginiaBalseiro) virginiabalseiro@gmail.com, [Alain Bourgeois](https://gitter.im/bourgeoa) alain.bourgeois10@gmail.com, [Sarven Capadisli](https://gitter.im/csarven). Committee members will recuse
```

File: code-of-conduct.md [hunk 3]
```diff
@@ -27,16 +27,16 @@ Best practices for the Solid Community include:
...
+themselves if they are the alleged harasser or the victim of a private harassment incident.
+- [Submit an issue](https://github.com/solid/culture/issues/new) to the solid/culture GitHub repository.
```
Comment (csarven) [inline code-of-conduct.md:38]:
```suggestion
- [Submit an issue](https://github.com/solid/process/issues/new) to the solid/process GitHub repository.
```

File: code-of-conduct.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ Participants in the Solid community are expected to behave professionally and re
...
+- Disagreements, both social and technical, happen all the time. Whenever inappropriate behaviors are observed members of the community should strive to bring the discussion back to a more professional level. In most cases misunderstandings and disagreements can be resolved informally. Do not hesitate to any CoC committee member if you feel that your concerns are not being sufficiently addressed. We hope that most concerns can be solved this way. However, if one person believes another's behavior is inappropriate (inconsistent with the Code of Conduct or the good working of the group), and ordinary communication between them is not possible, escalation of the issue can take place by contacting any CoC committee member. They may temporarily or permanently ban an individual from participating in activities, e.g. in a chat, mail or in Git for acting against the Code of Conduct.
```
Comment (csarven) [inline code-of-conduct.md:17]:
```suggestion
- Disagreements, both social and technical, happen all the time. Whenever inappropriate behaviors are observed members of the community should strive to bring the discussion back to a more professional level. In most cases misunderstandings and disagreements can be resolved informally. Do not hesitate to contact any CoC committee member if you feel that your concerns are not being sufficiently addressed. We hope that most concerns can be solved this way. However, if one person believes another's behavior is inappropriate (inconsistent with the Code of Conduct or the good working of the group), and ordinary communication between them is not possible, escalation of the issue can take place by contacting any CoC committee member. They may temporarily or permanently ban an individual from participating in activities, e.g. in a chat, mail or in Git for acting against the Code of Conduct.
```

**Review by csarven — CHANGES_REQUESTED** [review]
Do not link to hackmd for the minutes because that's unofficial and potentially ephemeral, and also not under the control of the Solid project. Refer only to specific minutes and topic URLs that's published from the https://github.com/solid/team/ repository.

---

I've requested a change to include myself in the CoC committee. I am a member of the Solid Team/DEIT, and the Solid CG Chair.

File: code-of-conduct.md (hunk 2 above)
Comment (theRealImy) [reply code-of-conduct.md:36]: @csarven please add your email too.

**Review by csarven — APPROVED** [review]

**Review by VirginiaBalseiro — APPROVED** [review]

**Review by justinwb — APPROVED** [review]
Many thanks to those volunteering their time to the CoC committee! ✅

**Pull request merged**